connector/Dockerfile

```diff
@@ -35,6 +36,7 @@ ENV WEB_HTTP_DATA_PORT="9191"
 ENV WEB_HTTP_DATA_PATH="/api/v1/data"
 ENV WEB_HTTP_IDS_PORT="8282"
 ENV WEB_HTTP_IDS_PATH="/api/v1/ids"
+ENV EDC_LAST_COMMIT_INFO_ENV=$EDC_LAST_COMMIT_INFO
```

Comment:
The ENV Var will be adjustable on container startup, via docker compose, etc., so it's the (more) true API.

So it should be the other way around, with the ENV var not having the ENV suffix and the build arg having the ARG suffix and an explanation why it even exists.